api/src/main/java/com/epam/pipeline/security/UserAccessService.java

```diff
@@ -74,10 +74,11 @@ public UserContext parseUser(final String userName,
 
     public UserContext getJwtUser(final JwtRawToken jwtRawToken, final JwtTokenClaims claims) {
         final UserContext jwtUser = new UserContext(jwtRawToken, claims);
+        final PipelineUser pipelineUser = userManager.loadUserByName(jwtUser.getUsername());
```

Comment:
Updating user for each JWT requests may produce some performance issues, let's move this logic under `validateUser` code flow (below if). User login date shall be updated only if difference between current timestamp and previous last login exceeds some threshold set in SystemPreferences (default value is 1 hour).